src/clj/witan/send/goal_seek.clj

```diff
@@ -98,14 +102,13 @@
   (mc/generate-configs (create-transition-modifier-seq m start end step)
                        config))
 
-(defn get-baseline-population [base-config year state-map]
-  (->> (str (get-in base-config [:output-parameters :output-dir]) "/Output_State_pop_only.csv")
-       csv->state-pop
-       (filter #(and (= (:calendar-year %) year)
-                     (every? identity (witan.send.model.prepare/test-predicates % state-map))))
-       (map #(select-keys % [:population]))
-       (apply merge-with +)
-       :population))
+(defn get-baseline-population [baseline year state-map]
+  (let [result (->> baseline
+                    (filter #(and (= (:calendar-year %) year)
+                                  (every? identity (witan.send.model.prepare/test-predicates % state-map))))
```

Comment:
This feels like the important line I need to understand and I don't. I'm not sure why you are selecting things here to be summed later.